src/libs/actions/PersonalDetails.js

```diff
@@ -130,16 +133,8 @@ function fetch() {
 
             // Set my personal details so they can be easily accessed and subscribed to on their own key
             Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS, myPersonalDetails);
-
-            // Get the timezone and put it in Onyx
-            fetchTimezone();
```

Comment:
Is this change right? I think maybe we still want to get the timezone? When we fetch the personal details initially or is that wrong?

Reply:
I think that was wrong too... why would we do that? They also both update at the same time now, so it is basically the same, except with less repeated requests.

Reply:
Hmm ok we always call the `fetchPersonalDetails()` here and before we were also calling this? 

https://github.com/Expensify/Expensify.cash/blob/195f58ded22ebc6c59632cd98ef4ed332fe17713/src/pages/home/HomePage.js#L77-L91

So, if we remove this will it take 30 minutes to fetch the timezone? Or not?

Reply:
Oh, good point, setInterval does not run immediately, adding a call when this loads so we get it immediately and then renew every 30m.